app/controllers/geocoder_controller.rb

```diff
@@ -9,14 +9,17 @@ class GeocoderController < ApplicationController
   authorize_resource :class => false
 
   def search
-    @params = normalize_params
+    normalize_params
```

Comment:
This could be a `before_action` now if we don't need to use the result?

Reply:
moved `normalize_params` to `before_action`